bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadOp.java

```diff
@@ -563,6 +565,10 @@ public long getLastAddConfirmed() {
         }
     }
 
+    public static ReadContext createReadContext(int bookieIndex, BookieId to, LedgerEntryRequest entry) {
```

Comment:
Don't need to be a public method? Looks only used for tests.

Reply:
Could we move this method into test?